Sources/TokamakStaticHTML/Views/Text/Text.swift

```diff
@@ -158,6 +158,35 @@ extension Text: AnyHTML {
   }
 }
 
+@_spi(TokamakStaticHTML) extension Text: HTMLConvertible {
+  @_spi(TokamakStaticHTML) public var innerHTML: String? {
+    let proxy = _TextProxy(self)
+    let innerHTML: String
+    switch proxy.storage {
+    case let .verbatim(text):
+      innerHTML = proxy.environment.domTextSanitizer(text)
+    case let .segmentedText(segments):
+      innerHTML = segments
+        .map {
+          TextSpan(
+            content: proxy.environment.domTextSanitizer($0.0.rawText),
+            attributes: Self.attributes(
+              from: $0.1,
+              environment: proxy.environment
+            )
+          )
+          .outerHTML(shouldSortAttributes: false, children: [])
+        }
+        .reduce("", +)
```

Comment:
I have a feeling that `reduce` with `+` may be slower than `reduce(into:)` with `Array.append`, but I haven't done any benchmarks. The thinking is that `+` causes heap allocations for newly created result strings, while `append` reuses the original string. Feel free to ignore this suggestion if you're not inclined to benchmark and micro-optimize this.

Reply:
I implemented this change to go from a `map` + `reduce("", +)` to a single `reduce(into:)`, but I haven't benchmarked it.

Reply:
has that changed been pushed? I don't see it here in the diff

Reply:
Ah, sorry. Only did it in one place.